src/casp/services/db/models/cells.py

```diff
@@ -43,26 +43,26 @@ class CellInfo(db.Base):
     # General Information
     cas_cell_index = sa.Column(sa.Integer, unique=True, nullable=False)
     cas_ingest_id = sa.Column(
-        sa.String(255), sa.ForeignKey(f"{CellIngestInfo.__tablename__}.cas_ingest_id"), nullable=False
+        sa.String(255), sa.ForeignKey(f"{CellIngestInfo.__tablename__}.cas_ingest_id"), nullable=False, index=True
     )
     cas_ingest = relationship("CellIngestInfo", backref=backref("cell_info", uselist=False))
     original_cell_id = sa.Column(sa.String(255), nullable=False)
     obs_metadata_extra = sa.Column(sa.JSON, nullable=False, default={})
-    is_primary_data = sa.Column(sa.Boolean(), default=True, nullable=True)
+    is_primary_data = sa.Column(sa.Boolean(), default=True, nullable=True, index=True)
     donor_id = sa.Column(sa.String(255), nullable=True)
     # Cell Features
-    cell_type = sa.Column(sa.String(255), nullable=False)
-    assay = sa.Column(sa.String(255), nullable=True)
-    development_stage = sa.Column(sa.String(255), nullable=True)
-    tissue = sa.Column(sa.String(255), nullable=True)
-    disease = sa.Column(sa.String(255), nullable=True)
-    organism = sa.Column(sa.String(255), nullable=True)
-    self_reported_ethnicity = sa.Column(sa.String(255), nullable=True)
-    sex = sa.Column(sa.String(255), nullable=True)
-    suspension_type = sa.Column(sa.String(255), nullable=True)
-    total_mrna_umis = sa.Column(sa.Integer, nullable=True)
+    cell_type = sa.Column(sa.String(255), nullable=False, index=True)
```

Comment:
Why do you need all of these indexes?  That seems like it's going to take a ton of space and I'm not sure you need them?

Reply:
I added indexes to all the columns that I want to have filters by in the admin page. Otherwise querying from admin becomes slow.

Reply:
Hmm...I'm wondering if it's worth it.  Do we really need that view in the admin UI and does it need to be performant?

Reply:
Well, I kinda think it is worth it. I myself will use this for querying the database from admin and explore it, or using raw sql. Especially, if we're moving to the "postgres first" idea, I've been making those queries a lot in bigquery. Also I think it can be very helpful to other ppl from Cellarium. 

Also, all these additional indexes together take about 5.2 gb space. The the data itself takes about 52gb. So it's 10% of the data size.

Reply:
Are there any performance concerns?

Reply:
fair enough!  The performance concerns are for loading the data.  I think with the pipeline that loads these, we'll want to disable the indexes, load, then reenable the indexes (generally good practice for bigger loads)

Reply:
Yep, that's exactly right